Dockerfile.build-base

```diff
@@ -2,24 +2,15 @@ FROM debian:jessie
 RUN apt-get update && \
     apt-get -y dist-upgrade && \
     apt-get -y install locales sudo vim less curl wget git rsync build-essential syslinux isolinux xorriso \
-        libblkid-dev libmount-dev libselinux1-dev genisoimage qemu-kvm python-pip
-RUN ln -s /usr/bin/genisoimage /usr/bin/mkisofs
+        libblkid-dev libmount-dev libselinux1-dev cpio python-pip ca-certificates
 RUN locale-gen en_US.UTF-8
-RUN curl -sSL https://get.docker.com/ | sh
-
-#ENV LANG en_US.UTF-8
-#ENV LANGUAGE en_US:en
-#ENV LC_ALL en_US.UTF-8
-#ENV TERM linux
+ADD https://get.docker.com/builds/Linux/x86_64/docker-1.8.2 /usr/bin/docker
```

Comment:
I don't like ADD-ing a URL because it the caching is slow, requires you to download each time.  Rather just do `curl -sL https://.../ > /usr/bin/docker`